js/data.js

```diff
@@ -339,6 +352,19 @@ var data = {
 		dozent: ["Prof. Dr. Christoph Meinel", "Dr. Michael Menzel"],
 		nameLV: "Dienstbasierte Webarchitekturen mit HTML5 und REST"
 	},
+	einfuehrungindiealgorithmik: {
+		kurz: "Algorithmik<br />Einführung",
+		lehrform: ["Vorlesung", "Übung"],
+		modul: ["Vertiefungsgebiete"],
+		semester: [ws15_16],
+		pflicht: false,
+		empfohlen: "",
+		vertiefung: ["IST", "OSIS", "SAMT"],
+		cp: 6,
+		vorher: [], //mathe1?
```

Comment:
The `vorher` is only used for strict rules. I think no one forces you to do math 1 before this course, even though it may be recommended.